README.md

```diff
@@ -16,7 +16,17 @@ Most generator code lives at [`src/lib`] and[`src/routes/beta`]. Some files used
 [`src/routes/beta`]: https://github.com/rianadon/Cosmos-Keyboards/tree/main/src/routes/beta
 [`src/model_gen`]: https://github.com/rianadon/Cosmos-Keyboards/tree/main/src/model_gen
 
-To run the generator locally, you'll need to [clone] the repository and [have installed Node.js][nodejs]. Then run these commands on the command line:
+To run the generator locally, you have two options:
+
+1. **Docker Setup** (Recommended for a quick start):
+   - Ensure you have Docker and Docker compose installed.
+   - Clone the repository.
+   - Run `docker compose up` in the project root directory.
+     - To avoid running the make scripts next build, set `SKIP_MAKE_FILES=true` in your .env file. This can reduce build time if all target files and docs have already been created.
```

Comment:
There are two main commands `docker compose up` and `docker compose build`. `build` is what happens inside of Dockerfile to create the container. Then `up` is starting the container up, when is when entrypoint.sh runs. If the container has not been built when `up` is called, then it will build. Otherwise it will launch the container and immediately run entrypoint.sh.

Reply:
You can also `docker compose up --build` to up and build.

Reply:
Ah I see. It behaves like the plain docker executable. Don't mind my comment then.